ansible/ec2.ini

```diff
@@ -39,7 +39,7 @@ vpc_destination_variable = private_ip_address
 
 # To tag instances on EC2 with the resource records that point to them from
 # Route53, uncomment and set 'route53' to True.
-route53 = False
+route53 = True
```

Comment:
hmm, it turns out I didn't need this after all. Might leave it in though, seems useful? 🤔